File: app/src/main/java/com/example/unimind/data/Alternativa.kt
package com.example.unimind.data

import com.google.gson.annotations.SerializedName

data class Alternativa(
    @SerializedName("idAlternativa")
    val idAlternativa: Int,

    @SerializedName("texto")
    val texto: String,

    @SerializedName("correta")
    val correta: Boolean,

    @SerializedName("idQuestao")
    val idQuestao: Int
)

File: Alternativa.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

[Table("Alternativa", Schema = "unimind")]
public class Alternativa
{
    [Key]
    [Column("idAlternativa")]
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public int IdAlternativa { get; set; }

    [Column("texto")]
    public string Texto { get; set; }

    [Column("correta")]
    public bool Correta { get; set; }

    [Column("idQuestao")]
    public int IdQuestao { get; set; }

    [JsonIgnore]
    [ForeignKey("IdQuestao")]
    public virtual Questao? Questao { get; set; }
}